release notes/v0.36.0.md

```diff
@@ -0,0 +1,76 @@
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### K6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, the configuration K6 derived from the execution context, and will also expose the `maxVUs` and `totalDuration` fields.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force K6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Bugs Fixed!
+
+* We had a long-standing issue ([#1771](https://github.com/grafana/k6/issues/1771)) making opening large files in the context of a K6 script somewhat unreliable under specific conditions in the past. This has now been addressed by [#2314](https://github.com/grafana/k6/pull/2314).
+* We addressed an issue uncovered by [our community](https://community.k6.io/t/v0-35-0-grpc-server-reflection-error/2383), which kept our users using GRPC to define multiple services in a single *proto* file. This issue was addressed by [#2265](https://github.com/grafana/k6/pull/2265).
+* Thanks to the contribution of [@Resousse](https://github.com/Resousse), we've now updated K6's `go-ntlmssp` dependency. The updating PR [#2290](https://github.com/grafana/k6/pull/2290) indeed fixes issues with NTLM Authentication backends returning two authorization headers. 
+
+## Maintenance
+
+- `v0.36.0` marks the switch to a new JS module's internal API. All our official JS modules have now been ported to this new version, which should make the process of developing modules easier and more streamlined in the future. See for more details:  [#2243](https://github.com/grafana/k6/pull/2243), [#2241](https://github.com/grafana/k6/pull/2241), [#2239](https://github.com/grafana/k6/pull/2239), [#2242](https://github.com/grafana/k6/pull/2242), [#2226](https://github.com/grafana/k6/pull/2226), and [#2232](https://github.com/grafana/k6/pull/2232).
```

Comment:
```suggestion
- `v0.36.0` marks the switch to a new JS module's internal API. Most of our official JS modules have now been ported to this new version, which should make the process of developing modules easier and more streamlined in the future. See for more details:  [#2243](https://github.com/grafana/k6/pull/2243), [#2241](https://github.com/grafana/k6/pull/2241), [#2239](https://github.com/grafana/k6/pull/2239), [#2242](https://github.com/grafana/k6/pull/2242), [#2226](https://github.com/grafana/k6/pull/2226), and [#2232](https://github.com/grafana/k6/pull/2232).
```
`grpc` hasn't been migrated yet

Reply:
Hehe, I was hesitant indeed, and didn't go to verify all of them one by one. Thanks for the heads up 🙇🏻